layouts/partials/cards/post.html

```diff
@@ -12,12 +12,18 @@ <h5 class="card-title">{{ .Title }}</h5>
         {{ end }}
       </div>
       <div class="card-footer">
-        <span class="float-left">{{ .Date.Format "January 2, 2006" }}{{ if site.Params.features.readingTime }} | {{ .ReadingTime }} minute{{ if (ne .ReadingTime 1) }}s{{ end }}{{ end }}</span>
+        <span class="float-left">
+            {{ .Date | time.Format ":date_full" }}
+            {{ if site.Params.features.readingTime }} | {{ .ReadingTime }} 
+              {{ if (ne .ReadingTime 1) }}
+                {{ i18n "minutes" }}
+              {{ else }}
+                {{ i18n "minute" }}
+              {{ end }}
```

Comment:
We don't need to check for singular or plural. We can just pass the count to `i18n` function and it will automatically chose singular or plural based on that count. For example:
```
{{ i18n "minute" .ReadingTime }}
```